test/test_torch.py

```diff
@@ -17489,6 +17493,52 @@ def test_hypot(self, device, dtype):
                 expected = np.hypot(input[0].cpu().numpy(), input[1].cpu().numpy())
             self.assertEqual(actual, expected)
 
+    @dtypesIfCPU(torch.bfloat16, torch.float32, torch.float64)
```

Comment:
You also support half on CUDA, so I'd expected `@dtypesIfCUDA(torch.half, ...)` as well.